### PR TITLE
Feature ab101 add remove player buttons

### PR DESCRIPTION
**Pull request by who-sane**

### CODE REVIEW

**Review by EveJoyceStudent — DISMISSED** [review]
Don't merge without a second review

**Review by EveJoyceStudent — COMMENTED** [review]
Happy with this but would prefer another review as well

**Review by rikichoi — APPROVED** [review]
looks good :D

**Pull request merged**
